GoogleDataTransport/GDTCCTTests/Common/TestStorage/GDTCCTTestStorage.m

```diff
@@ -207,10 +207,14 @@ - (void)defaultBatchWithEventSelector:(nonnull GDTCORStorageEventSelector *)even
       }];
 }
 
-- (FBLPromise<GDTCORLibraryData> *)
-    fetchAndUpdateLibraryDataForKey:(NSString *)key
-                              klass:(Class)klass
-                     readWriteBlock:(GDTCORStorageLibraryDataReadWriteBlock)readWriteBlock {
+- (FBLPromise<NSNull *> *)fetchAndUpdateClientMetricsWithReadWriteBlock:
+    (GDTCORMetricsMetadata *_Nullable (^)(GDTCORMetricsMetadata *_Nullable fetchedMetadata,
+                                          NSError *_Nullable fetchError))readWriteBlock {
```

Comment:
fetchAndUpdateClientMetricsWithHandler?

Reply:
Sure, I'll ensure it is fully documented.

Done in 1c37aa6